### DISCUSSION TITLE
Difference between this release 0.5.4 and the newer one 0.7.5

### DISCUSSION THREAD

**Discussion by cheadrian** [twoyi/twoyi#132] [Q&A]
Are there updates in the [CoolAPK 0.7.5](https://www.coolapk.com/apk/io.twoyi) version? It seems that they aren't public, as the version on Github wasn't updated for 9 months.

**Comment by slopsommelier** [comment]
> Are there updates in the [CoolAPK 0.7.5](https://www.coolapk.com/apk/io.twoyi) version? It seems that they aren't public, as the version on Github wasn't updated for 9 months. 

Also want to know
